src/lib/features/feature-lifecycle/feature-lifecycle-count-controller.ts

```diff
@@ -0,0 +1,80 @@
+import {
+    type IFeatureLifecycleReadModel,
+    type IUnleashConfig,
+    type IUnleashServices,
+    NONE,
+} from '../../types';
+import type { OpenApiService } from '../../services';
+import { createResponseSchema, getStandardResponses } from '../../openapi';
+import Controller from '../../routes/controller';
+import type { Request, Response } from 'express';
+import {
+    type FeatureLifecycleCountSchema,
+    featureLifecycleCountSchema,
+} from '../../openapi/spec/feature-lifecycle-count-schema';
+
+export default class FeatureLifecycleCountController extends Controller {
+    private featureLifecycleReadModel: IFeatureLifecycleReadModel;
+
+    private openApiService: OpenApiService;
+
+    constructor(
+        config: IUnleashConfig,
+        {
+            openApiService,
+            featureLifecycleReadModel,
+        }: Pick<
+            IUnleashServices,
+            'openApiService' | 'featureLifecycleReadModel'
+        >,
+    ) {
+        super(config);
+        this.featureLifecycleReadModel = featureLifecycleReadModel;
+        this.openApiService = openApiService;
+
+        this.route({
+            method: 'get',
+            path: '/count',
+            handler: this.getStageCount,
+            permission: NONE,
+            middleware: [
+                openApiService.validPath({
+                    tags: ['Unstable'],
+                    summary: 'Get all features lifecycle stage count',
+                    description:
+                        'Information about the number of features in each lifecycle stage.',
+                    operationId: 'getFeatureLifecycleStageCount',
+                    responses: {
+                        200: createResponseSchema(
+                            'featureLifecycleCountSchema',
+                        ),
+                        ...getStandardResponses(401, 403, 404),
+                    },
+                }),
+            ],
+        });
+    }
+
+    async getStageCount(
```

Comment:
I'm not sure it has to be memoized, but I didn't check how expensive this query is

Reply:
i was checking the explain plan and it was very cheap. I can add query tracking in the next PR if we find in the UI that those are slow. Also we can set infrequent interval in the SWR hook since this doesn't need to update very often